# Hydrolysis Reactions (HRs) Family 4 / Subfamily 1

<!--[TOC]在这里不可以用-->

<ul id="myTab" class="nav nav-tabs">
  <!-- active 指的是默认页 -->
  <li class="active">
    <!-- herf中名字于下文id对应 -->
    <!-- 这里只改herf和tab1 -->
    <a href="#tab1" data-toggle="tab">A0A0E3TKF2</a>
  </li>
  <li><a href="#tab2" data-toggle="tab">I5AX48</a></li>
</ul>
<div id="myTabContent" class="tab-content" markdown="1">
  <!-- 此处的id与上文herf对应 其他的不要改-->
  <div class="tab-pane fade in active" id="tab1" markdown="1">

<h2>Literature Information</h2>
<table>
<thead>
<tr>
<th style="text-align:left">Title</th>
<th style="text-align:left">Identification and functional expression of genes encoding flavonoid <em>O</em>‐ and <em>C</em>‐glycosidases in intestinal bacteria</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Author</td>
<td style="text-align:left">Annett Braune, Wolfram Engst, Michael Blaut</td>
</tr>
<tr>
<td style="text-align:left">DOI</td>
<td style="text-align:left"><a href="https://doi.org/10.1111/1462-2920.12864">10.1111/1462-2920.12864</a></td>
</tr>
<tr>
<td style="text-align:left">Abstract</td>
<td style="text-align:left">Gut bacteria play a crucial role in the metabolism of dietary flavonoids and thereby influence the bioactivity of these compounds in the host. The intestinal <em>Lachnospiraceae</em> strain CG19‐1 and <em>Eubacterium cellulosolvens</em> are able to deglycosylate <em>C</em>‐ and <em>O</em>‐coupled flavonoid glucosides. Growth of strain CG19‐1 in the presence of the isoflavone <em>C</em>‐glucoside puerarin (daidzein 8‐<em>C</em>‐glucoside) led to the induction of two proteins (DfgC, DfgD). Heterologous expression of the encoding genes (<em>dfgC</em>, <em>dfgD</em>) in <em>Escherichia coli</em> revealed no <em>C</em>‐deglycosylating activity in the resulting cell extracts but cleavage of flavonoid <em>O</em>‐glucosides such as daidzin (daidzein 7‐<em>O</em>‐glucoside). The recombinant DfgC and DfgD proteins were purified and characterized with respect to their quaternary structure, substrate and cofactor specificity. The products of the corresponding genes (<em>dfgC</em>, <em>dfgD</em>) from <em>E. cellulosolvens</em> also catalysed the <mark><em>O</em>‐deglycosylation of daidzin</mark> following their expression in <em>E. coli</em>. In combination with three recombinant proteins encoded by adjacent genes in <em>E. cellulosolvens</em> (<em>dfgA</em>, <em>dfgB</em>, <em>dfgE</em>), DfgC and DfgD from <em>E</em>*. cellulosolvens* catalysed the deglycosylation of the flavone <em>C</em>‐glucosides homoorientin (luteolin 6‐<em>C</em>‐glucoside) and isovitexin (apigenin 6‐<em>C</em>‐glucoside). Even intact cells of <em>E</em>*. coli* expressing the five <em>E. cellulosolvens</em> genes cleaved these flavone <em>C</em>‐glucosides and, also, flavonoid <em>O</em>‐glucosides to the corresponding aglycones.</td>
</tr>
</tbody>
</table>
<h2>Experimental results</h2>
<ul>
<li><strong>Enzyme</strong></li>
</ul>
<p>Uniprot ID: <a href="https://www.uniprot.org/uniprot/A0A0E3TKF2">A0A0E3TKF2</a></p>
<p>Protein: DfgC</p>
<p>Organism: <em>Catenibacillus scindens</em></p>
<p>Length: 279 AA</p>
<p>Taxonomic identifier: <a href="https://www.uniprot.org/taxonomy/673271">673271</a> [<a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&amp;id=673271">NCBI</a>]</p>
<ul>
<li><strong>Pfam</strong></li>
</ul>
<table>
<thead>
<tr>
<th>Source</th>
<th>Domain</th>
<th>Start</th>
<th>End</th>
<th>E-value (Domain)</th>
<th>Coverage</th>
</tr>
</thead>
<tbody>
<tr>
<td>Pfam-A</td>
<td>AP_endonuc_2</td>
<td>30</td>
<td>204</td>
<td>1.1e-18</td>
<td>0.914</td>
</tr>
</tbody>
</table>
<p>Program: <code>hmmscan</code></p>
<p>Version: 3.1b2 (February 2015)</p>
<p>Method: <code>hmmscan --domtblout hmmscan.tbl --noali -E 1e-5 pfam query.fa </code></p>
<p>Date: Mon Jul 20 14:32:16 2020</p>
<p>Description:</p>
<p>AP_endonuc_2</p>
<p><a href="https://pfam.xfam.org/family/AP_endonuc_2"><strong>Pfam</strong></a></p>
<p>This TIM alpha/beta barrel structure is found in xylose isomerase (<a href="http://www.uniprot.org/entry/P19148">P19148</a>) and in endonuclease IV (<a href="http://www.uniprot.org/entry/P12638">P12638</a> EC:3.1.21.2). This domain is also found in the N termini of bacterial myo-inositol catabolism proteins. These are involved in the myo-inositol catabolism pathway, and is required for growth on myo-inositol in Rhizobium leguminosarum bv. viciae<sup class="footnote-ref"><a href="#fn1" id="fnref1">[1]</a></sup>.</p>
<p><a href="http://www.ebi.ac.uk/interpro/entry/InterPro/IPR013022/"><strong>InterPro</strong></a></p>
<p>This entry represents a structural motif with a beta/alpha TIM barrel found in several proteins families:</p>
<p>▹ Endonuclease IV (<a href="http://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.1.21.2">3.1.21.2</a>), an AP (apurinic/apyrimidinic) endonuclease that primes DNA repair synthesis by cleaving the DNA backbone 5’ of AP sites<sup class="footnote-ref"><a href="#fn2" id="fnref2">[2]</a></sup>.</p>
<p>▹ L-rhamnose isomerase (<a href="http://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=5.3.1.14">5.3.1.14</a>), a tetramer of four TIM barrels that catalyses the isomerisation between aldoses and ketoses, such as between L-rhamnose and L-rhamnulose<sup class="footnote-ref"><a href="#fn3" id="fnref3">[3]</a></sup>.</p>
<p>▹ Xylose isomerase (<a href="http://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=5.3.1.5">5.3.1.5</a>), which catalyses the first reaction in the catabolism of D-xylose by converting D-xylose to D-xylulose<sup class="footnote-ref"><a href="#fn4" id="fnref4">[4]</a></sup>.</p>
<p>▹ Mannonate dehydratase UxuA, which along with mannonate oxidoreductase converts D-fructuronate to 2-keto-3-deoxy-D-gluconate<sup class="footnote-ref"><a href="#fn5" id="fnref5">[5]</a></sup>.</p>
<p>These proteins share similar, but not identical, metal-binding sites. In addition, xylose isomerase and L-rhamnose isomerase each have additional alpha-helical domains involved in tetramer formation.</p>
<ul>
<li><strong>Reaction</strong></li>
</ul>
<p><em>Recombined with <a href="https://www.uniprot.org/uniprot/A0A0E3TJD6">A0A0E3TJD6</a></em></p>
<p><a href="https://pubchem.ncbi.nlm.nih.gov/compound/107971">daidzin</a> ⇒ <a href="https://pubchem.ncbi.nlm.nih.gov/compound/daidzein">daidzein</a> + <a href="https://pubchem.ncbi.nlm.nih.gov/compound/glucose">glucose</a></p>
<figure>
<div class="linerow">
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/HR4/[L1]daidzin.png" alt="[L1]daidzin" style="width:100%">
  </div>
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/common_symbol/right_arrow.png" alt="right_arrow" style="width:100%">
  </div>
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/HR4/[R1]daidzein.png" alt="[R1]daidzein" style="width:100%">
  </div>
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/common_symbol/plus.png" alt="plus" style="width:100%">
  </div>
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/HR4/[R2]glucose.png" alt="[R2]glucose" style="width:100%">
  </div>
</div>
</figure>
<h2>References</h2>
<hr class="footnotes-sep" />
<section class="footnotes">
<ol class="footnotes-list">
<li id="fn1" class="footnote-item"><p>Fry J, Wood M, Poole P S. Investigation of myo-inositol catabolism in Rhizobium leguminosarum bv. viciae and its effect on nodulation competitiveness[J]. Molecular plant-microbe interactions, 2001, 14(8): 1016-1025. <a href="#fnref1" class="footnote-backref">↩︎</a></p>
</li>
<li id="fn2" class="footnote-item"><p>Hosfield D J, Guan Y, Haas B J, et al. Structure of the DNA repair enzyme endonuclease IV and its DNA complex: double-nucleotide flipping at abasic sites and three-metal-ion catalysis[J]. Cell, 1999, 98(3): 397-408. <a href="#fnref2" class="footnote-backref">↩︎</a></p>
</li>
<li id="fn3" class="footnote-item"><p>Yoshida H, Yamada M, Ohyama Y, et al. The structures of L-rhamnose isomerase from Pseudomonas stutzeri in complexes with L-rhamnose and D-allose provide insights into broad substrate specificity[J]. Journal of molecular biology, 2007, 365(5): 1505-1516. <a href="#fnref3" class="footnote-backref">↩︎</a></p>
</li>
<li id="fn4" class="footnote-item"><p>Meilleur F, Snell E H, Van der Woerd M J, et al. A quasi-Laue neutron crystallographic study of D-xylose isomerase[J]. European Biophysics Journal, 2006, 35(7): 601-609. <a href="#fnref4" class="footnote-backref">↩︎</a></p>
</li>
<li id="fn5" class="footnote-item"><p>Robert-Baudouy J, Portalier R, Stoeber F. Regulation of hexuronate system genes in Escherichia coli K-12: multiple regulation of the uxu operon by exuR and uxuR gene products[J]. Journal of Bacteriology, 1981, 145(1): 211-220. <a href="#fnref5" class="footnote-backref">↩︎</a></p>
</li>
</ol>
</section>

  </div>
  <div class="tab-pane fade" id="tab2" markdown="1">

<h2>Literature Information</h2>
<table>
<thead>
<tr>
<th style="text-align:left">Title</th>
<th style="text-align:left">Identification and functional expression of genes encoding flavonoid <em>O</em>‐ and <em>C</em>‐glycosidases in intestinal bacteria</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Author</td>
<td style="text-align:left">Annett Braune, Wolfram Engst, Michael Blaut</td>
</tr>
<tr>
<td style="text-align:left">DOI</td>
<td style="text-align:left"><a href="https://doi.org/10.1111/1462-2920.12864">10.1111/1462-2920.12864</a></td>
</tr>
<tr>
<td style="text-align:left">Abstract</td>
<td style="text-align:left">Gut bacteria play a crucial role in the metabolism of dietary flavonoids and thereby influence the bioactivity of these compounds in the host. The intestinal <em>Lachnospiraceae</em> strain CG19‐1 and <em>Eubacterium cellulosolvens</em> are able to deglycosylate <em>C</em>‐ and <em>O</em>‐coupled flavonoid glucosides. Growth of strain CG19‐1 in the presence of the isoflavone <em>C</em>‐glucoside puerarin (daidzein 8‐<em>C</em>‐glucoside) led to the induction of two proteins (DfgC, DfgD). Heterologous expression of the encoding genes (<em>dfgC</em>, <em>dfgD</em>) in <em>Escherichia coli</em> revealed no <em>C</em>‐deglycosylating activity in the resulting cell extracts but cleavage of flavonoid <em>O</em>‐glucosides such as daidzin (daidzein 7‐<em>O</em>‐glucoside). The recombinant DfgC and DfgD proteins were purified and characterized with respect to their quaternary structure, substrate and cofactor specificity. The products of the corresponding genes (<em>dfgC</em>, <em>dfgD</em>) from <em>E. cellulosolvens</em> also catalysed the <mark><em>O</em>‐deglycosylation of daidzin</mark> following their expression in <em>E. coli</em>. In combination with three recombinant proteins encoded by adjacent genes in <em>E. cellulosolvens</em> (<em>dfgA</em>, <em>dfgB</em>, <em>dfgE</em>), DfgC and DfgD from <em>E</em>*. cellulosolvens* catalysed the deglycosylation of the flavone <em>C</em>‐glucosides homoorientin (luteolin 6‐<em>C</em>‐glucoside) and isovitexin (apigenin 6‐<em>C</em>‐glucoside). Even intact cells of <em>E</em>*. coli* expressing the five <em>E. cellulosolvens</em> genes cleaved these flavone <em>C</em>‐glucosides and, also, flavonoid <em>O</em>‐glucosides to the corresponding aglycones.</td>
</tr>
</tbody>
</table>
<h2>Experimental results</h2>
<ul>
<li><strong>Enzyme</strong></li>
</ul>
<p>Uniprot ID: <a href="https://www.uniprot.org/uniprot/I5AX48">I5AX48</a></p>
<p>Protein: Sugar phosphate isomerase/epimerase</p>
<p>Organism: <em>[Eubacterium] cellulosolvens 6</em></p>
<p>Length: 278 AA</p>
<p>Taxonomic identifier: <a href="https://www.uniprot.org/taxonomy/633697">633697</a> [<a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&amp;id=633697">NCBI</a>]</p>
<ul>
<li><strong>Pfam</strong></li>
</ul>
<table>
<thead>
<tr>
<th>Source</th>
<th>Domain</th>
<th>Start</th>
<th>End</th>
<th>E-value (Domain)</th>
<th>Coverage</th>
</tr>
</thead>
<tbody>
<tr>
<td>Pfam-A</td>
<td>AP_endonuc_2</td>
<td>29</td>
<td>206</td>
<td>1.4e-19</td>
<td>0.983</td>
</tr>
</tbody>
</table>
<p>Program: <code>hmmscan</code></p>
<p>Version: 3.1b2 (February 2015)</p>
<p>Method: <code>hmmscan --domtblout hmmscan.tbl --noali -E 1e-5 pfam query.fa </code></p>
<p>Date: Mon Jul 20 14:32:16 2020</p>
<p>Description:</p>
<p>AP_endonuc_2</p>
<p><a href="https://pfam.xfam.org/family/AP_endonuc_2"><strong>Pfam</strong></a></p>
<p>This TIM alpha/beta barrel structure is found in xylose isomerase (<a href="http://www.uniprot.org/entry/P19148">P19148</a>) and in endonuclease IV (<a href="http://www.uniprot.org/entry/P12638">P12638</a> EC:3.1.21.2). This domain is also found in the N termini of bacterial myo-inositol catabolism proteins. These are involved in the myo-inositol catabolism pathway, and is required for growth on myo-inositol in Rhizobium leguminosarum bv. viciae<sup class="footnote-ref"><a href="#fn1" id="fnref1">[1]</a></sup>.</p>
<p><a href="http://www.ebi.ac.uk/interpro/entry/InterPro/IPR013022/"><strong>InterPro</strong></a></p>
<p>This entry represents a structural motif with a beta/alpha TIM barrel found in several proteins families:</p>
<p>▹ Endonuclease IV (<a href="http://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.1.21.2">3.1.21.2</a>), an AP (apurinic/apyrimidinic) endonuclease that primes DNA repair synthesis by cleaving the DNA backbone 5’ of AP sites<sup class="footnote-ref"><a href="#fn2" id="fnref2">[2]</a></sup>.</p>
<p>▹ L-rhamnose isomerase (<a href="http://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=5.3.1.14">5.3.1.14</a>), a tetramer of four TIM barrels that catalyses the isomerisation between aldoses and ketoses, such as between L-rhamnose and L-rhamnulose<sup class="footnote-ref"><a href="#fn3" id="fnref3">[3]</a></sup>.</p>
<p>▹ Xylose isomerase (<a href="http://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=5.3.1.5">5.3.1.5</a>), which catalyses the first reaction in the catabolism of D-xylose by converting D-xylose to D-xylulose<sup class="footnote-ref"><a href="#fn4" id="fnref4">[4]</a></sup>.</p>
<p>▹ Mannonate dehydratase UxuA, which along with mannonate oxidoreductase converts D-fructuronate to 2-keto-3-deoxy-D-gluconate<sup class="footnote-ref"><a href="#fn5" id="fnref5">[5]</a></sup>.</p>
<p>These proteins share similar, but not identical, metal-binding sites. In addition, xylose isomerase and L-rhamnose isomerase each have additional alpha-helical domains involved in tetramer formation.</p>
<ul>
<li><strong>Reaction</strong></li>
</ul>
<p><em>Catalyse  reaction with expression of  <a href="https://www.uniprot.org/uniprot/I5AX46">I5AX46</a>/<a href="https://www.uniprot.org/uniprot/I5AX47">I5AX47</a>/<a href="https://www.uniprot.org/uniprot/I5AX48">I5AX48</a>/<a href="https://www.uniprot.org/uniprot/I5AX49">I5AX49</a>/<a href="https://www.uniprot.org/uniprot/I5AX50">I5AX50</a> simultaneously</em></p>
<p><a href="https://pubchem.ncbi.nlm.nih.gov/compound/107971">daidzin</a> ⇒ <a href="https://pubchem.ncbi.nlm.nih.gov/compound/daidzein">daidzein</a> + <a href="https://pubchem.ncbi.nlm.nih.gov/compound/glucose">glucose</a></p>
<figure>
<div class="linerow">
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/HR4/[L1]daidzin.png" alt="[L1]daidzin" style="width:100%">
  </div>
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/common_symbol/right_arrow.png" alt="right_arrow" style="width:100%">
  </div>
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/HR4/[R1]daidzein.png" alt="[R1]daidzein" style="width:100%">
  </div>
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/common_symbol/plus.png" alt="plus" style="width:100%">
  </div>
  <div class="linecolumn">
    <img src="../static/images/chemical_structure/HR4/[R2]glucose.png" alt="[R2]glucose" style="width:100%">
  </div>
</div>
</figure>
<h2>References</h2>
<hr class="footnotes-sep" />
<section class="footnotes">
<ol class="footnotes-list">
<li id="fn1" class="footnote-item"><p>Fry J, Wood M, Poole P S. Investigation of myo-inositol catabolism in Rhizobium leguminosarum bv. viciae and its effect on nodulation competitiveness[J]. Molecular plant-microbe interactions, 2001, 14(8): 1016-1025. <a href="#fnref1" class="footnote-backref">↩︎</a></p>
</li>
<li id="fn2" class="footnote-item"><p>Hosfield D J, Guan Y, Haas B J, et al. Structure of the DNA repair enzyme endonuclease IV and its DNA complex: double-nucleotide flipping at abasic sites and three-metal-ion catalysis[J]. Cell, 1999, 98(3): 397-408. <a href="#fnref2" class="footnote-backref">↩︎</a></p>
</li>
<li id="fn3" class="footnote-item"><p>Yoshida H, Yamada M, Ohyama Y, et al. The structures of L-rhamnose isomerase from Pseudomonas stutzeri in complexes with L-rhamnose and D-allose provide insights into broad substrate specificity[J]. Journal of molecular biology, 2007, 365(5): 1505-1516. <a href="#fnref3" class="footnote-backref">↩︎</a></p>
</li>
<li id="fn4" class="footnote-item"><p>Meilleur F, Snell E H, Van der Woerd M J, et al. A quasi-Laue neutron crystallographic study of D-xylose isomerase[J]. European Biophysics Journal, 2006, 35(7): 601-609. <a href="#fnref4" class="footnote-backref">↩︎</a></p>
</li>
<li id="fn5" class="footnote-item"><p>Robert-Baudouy J, Portalier R, Stoeber F. Regulation of hexuronate system genes in Escherichia coli K-12: multiple regulation of the uxu operon by exuR and uxuR gene products[J]. Journal of Bacteriology, 1981, 145(1): 211-220. <a href="#fnref5" class="footnote-backref">↩︎</a></p>
</li>
</ol>
</section>

  </div>
</div>